sites/partners/src/pages/api/adapter/[...backendUrl].ts

```diff
@@ -50,7 +50,20 @@ export default async (req: NextApiRequest, res: NextApiResponse) => {
     const cookies = await jar.getSetCookieStrings(process.env.BACKEND_API_BASE || "")
     res.setHeader("Set-Cookie", cookies)
     res.statusMessage = response.statusText
-    res.status(response.status).json(response.data)
+    if (response.headers) {
+      if (response.headers["content-type"]) {
+        res.setHeader("content-type", response.headers["content-type"])
+      }
+      if (response.headers["content-disposition"]) {
+        res.setHeader("content-disposition", response.headers["content-disposition"])
+      }
+      // TODO: figure out why this chunked encoding isn't working
+      // if (response.headers["transfer-encoding"]) {
+      //   res.setHeader("transfer-encoding", response.headers["transfer-encoding"])
+      //   res.removeHeader("content-length")
+      // }
+    }
+    res.status(response.status).send(response.data)
```

Comment:
When downloading a zip file it doesn't come back as json so additional fields need to be added. However there still seems to be some issues with the file when going through the adapter vs directly hitting the backend. More investigation is needed